README.md

```diff
@@ -8,20 +8,19 @@
 
 [![][appveyor img]][appveyor]
 [![][circleci img]][circleci]
-[![][cirrusci img]][cirrusci]
+[![CircleCI Status][circleci img]][circleci]
```

Comment:
<img width="637" height="902" alt="Image" src="https://github.com/user-attachments/assets/6b7acb0b-4fb3-4dca-a710-771b3c4583fa" />
Double circle CI. Please find how to avoid this